src/css/app.css

```diff
@@ -172,8 +172,6 @@
 
 .codebase-tree .namespace-tree .namespace-content {
   margin-left: 1rem;
-  transform-origin: left top;
-  animation: slide-down 0.2s ease-out;
```

Comment:
Remove the slide down animation of items in the sidebar; it never really felt right.